1-org/projects.tf

```diff
@@ -150,6 +150,6 @@ module "dns_hub" {
 
   labels = {
     environment      = "prod"
-    application_name = "prj-dns-hub"
+    application_name = "dns-hub"
```

Comment:
if the label is fixed, the search for the label needs to be fixed too: 
https://github.com/terraform-google-modules/terraform-example-foundation/blob/e7ef9c22071fd562aba400ff0026cfb60a738a30/3-networks/envs/shared/main.tf#L26

Reply:
Good catch; I'll fix those as well